src/webui/.editorconfig

```diff
@@ -0,0 +1,12 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
```

Comment:
And I wonder why your files do not have trailing new line even if this is set to `true`.